docs/api/en/renderers/WebGLRenderer.html

```diff
@@ -427,12 +427,11 @@ <h3>[method:null render]( [param:Scene scene], [param:Camera camera] )</h3>
 		<h3>[method:null renderBufferDirect]( [param:Camera camera], [param:Fog fog], [param:Geometry geometry], [param:Material material], [param:Object3D object], [param:Object group] )</h3>
 		<p>Render a buffer geometry group using the camera and with the specified material.</p>
 
-		<h3>[method:null renderBufferImmediate]( [param:Object3D object], [param:shaderprogram program], [param:Material shading] )</h3>
+		<h3>[method:null renderBufferImmediate]( [param:Object3D object], [param:shaderprogram program] )</h3>
 		<p>object - an instance of [page:Object3D]<br />
 		program - an instance of shaderProgram<br />
-		shading - an instance of Material<br /><br />
 
-		Render an immediate buffer. Gets called by renderImmediateObject.
+		Render an immediate buffer. Gets called by renderObjectImmediate.
```

Comment:
What is an immediate buffer?

Reply:
Maybe like so:
```
Renders an instance of [page:ImmediateRenderObject]. Gets called by renderObjectImmediate().
```

Reply:
Will do! I won't be able to apply this particular change in Chinese though.